src/App.tsx

```diff
@@ -1,12 +1,16 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import {
+  NavLink,
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import classNames from 'classnames';
+import { TabsPage } from './components/TabsPage/TabsPage';
+import { HomePage } from './components/HomePage';
```

Comment:
Add this pages to the PR
![image](https://github.com/mate-academy/react_tabs-with-router/assets/91411692/a7a6c10c-74bf-411e-89f2-b3aaebbc272a)